.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: rust
+# XXX: switch back to stable once Rust 1.41.0 is out
+rust: nightly-2019-12-01
```

Comment:
We use nightly until Rust 1.41 is released since the install-upgrade feature is stabilized there.

Reply:
Is this feature needed? Wouldn't `cmd || cargo install` enough?

Reply:
Then it will not update old mdbook in CI cache.
I think we could wait until next beta or next stable.

Reply:
Well it shouldn't ever update mdbook. mdbook needs to always match the Cargo.toml

Reply:
I meant both mdbook cli and mdbook lib.
And here in this PR I update lock file too.

Reply:
Oh I see what you mean now, I think it's also fine to not cache mdbook for now.